[2020-06-16 23:08:16,148] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: generator_v0.generate_file 2020-06-17T03:07:00+00:00 [queued]>
[2020-06-16 23:08:16,154] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: generator_v0.generate_file 2020-06-17T03:07:00+00:00 [queued]>
[2020-06-16 23:08:16,154] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-16 23:08:16,154] {taskinstance.py:880} INFO - Starting attempt 1 of 2
[2020-06-16 23:08:16,154] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-16 23:08:16,749] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): generate_file> on 2020-06-17T03:07:00+00:00
[2020-06-16 23:08:16,753] {standard_task_runner.py:53} INFO - Started process 3372 to run task
[2020-06-16 23:08:17,368] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: generator_v0.generate_file 2020-06-17T03:07:00+00:00 [running]> strx
[2020-06-16 23:08:17,379] {logging_mixin.py:112} INFO - Generating  1592363297 ...
[2020-06-16 23:08:17,385] {logging_mixin.py:112} INFO - Finished.
[2020-06-16 23:08:19,387] {python_operator.py:114} INFO - Done. Returned value was: _
[2020-06-16 23:08:20,086] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=generator_v0, task_id=generate_file, execution_date=20200617T030700, start_date=20200617T030816, end_date=20200617T030820
